src/core/backbone_router/bbr_leader.cpp

```diff
@@ -217,6 +217,14 @@ void Leader::UpdateBackboneRouterPrimary(void)
         state = kStateUnchanged;
     }
 
+    // Restrain the range of MLR timeout to be always valid
+    config.mMlrTimeout = config.mMlrTimeout < static_cast<uint32_t>(Mle::kMlrTimeoutMin)
+                             ? static_cast<uint32_t>(Mle::kMlrTimeoutMin)
+                             : config.mMlrTimeout;
+    config.mMlrTimeout = config.mMlrTimeout > static_cast<uint32_t>(Mle::kMlrTimeoutMax)
+                             ? static_cast<uint32_t>(Mle::kMlrTimeoutMax)
+                             : config.mMlrTimeout;
```

Comment:
May add log when `config.mMlrTimeout` is changed and different from the value in Thread Network Data?

Reply:
Added a warning log